src/components/DuosHeader.js

```diff
@@ -386,7 +386,7 @@ class DuosHeader extends Component {
       'data-tip': 'Need help? Contact us for some assistance', 'data-for': 'tip_requestAccess'
     }, [contactUsIcon, contactUsText]);
 
-    const supportrequestModal = SupportRequestModal({
+    const supportrequestModal = h(SupportRequestModal, {
```

Comment:
this line fixes the "unmounted component exception" that occasionally shows up

Reply:
hmm: I am still getting this error ??
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/17440010/185153276-ac071522-6432-40db-88f0-185d929a94bb.png">

Reply:
hopefully less at least... maybe this same issue is in multiple places :X